app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> bool:
+    if len(command.split()) != 3:
+        raise ValueError
+    cmd, source_file, new_file = command.split()
+    os.makedirs(os.path.dirname(os.path.abspath(new_file)),
+                exist_ok=True)
+    with open(source_file, "r") as file_in, open(new_file, "w") as file_out:
```

Comment:
this needs to be wrapped with try-except to catch any tracebacks related to file operations